moveit_ros/planning_interface/move_group_interface/include/moveit/move_group_interface/move_group_interface.h

```diff
@@ -135,6 +135,33 @@ class MoveGroupInterface
     double planning_time_;
   };
 
+  struct ManipulationPlan
+  {
+    /// The full starting state of the robot at the start of the trajectory
+    moveit_msgs::RobotState start_state_;
```

Comment:
ROS C++ style says no underscore after struct member variables

Reply:
I followed the convention that other public structs follows (e.g. `MoveGroupInterface::Options`).
Would you prefer to changes them all or structs I added only?

Reply:
I think following the conventions is fine here but we should make an issue to adapt ROS C++ style in the master branch.

Reply:
@aochiai can you change the other structs to follow the style guide correctly, in a separate PR?